n3fit/src/evolven3fit/__init__.py

```diff
@@ -0,0 +1 @@
+from . import evolven3fit_API
```

Comment:
I don't think I understand this import. Why is it useful?

Reply:
It is not :)